app/Domain/Rooms/Actions/GetAllRooms.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace App\Domain\Rooms\Actions;
+
+use App\Domain\Rooms\Models\Room;
+
+class GetAllRooms
+{
+    public function execute(): array
+    {
+        return Room::all()->toArray();
```

Comment:
Нужно возвращать коллекцию, а не массив. Иначе потом не возможно использовать ресурсы для ответов в контроллере. Т.е. должно быть просто
```suggestion
        return Room::all();
```